### ISSUE TITLE
Setup warnings on fresh install

### ISSUE THREAD

**Issue by arcooke** [freenas/iocage-plugin-nextcloud#7]
Hi.  I recently set up Nextcloud and noticed it gives a number of warnings and recommendations in Settings>Administration>Overview.  PHP environment variables seem to be the big one.  Is this something that should be fixed on the plugin installation?  I can go into the jail and manually tune these things but maybe they should be auto-configured on plugin installation

![image](https://user-images.githubusercontent.com/6278400/51496401-3e06ce00-1d85-11e9-8371-f4aeef63aa11.png)

**Comment by louizo** [comment]
Agree, I think it should be fix during installation. I'm currently working on this (just started)...
I will request merge for fix on php.ini first, if it's accepted will do other patches for more complex fix (such as imagick) and big int fixes

**Comment by arcooke** [comment]
Thanks!  I appreciate the quick response.  I'll keep an eye out for your changes so I can make matching updates to my existing installation.  I'm concerned about using the built-in updater tool while the environment variables are misconfigured.  I don't know if there's any risk but I'd rather not chance it

**Comment by miwi-fbsd** [comment]
@arcooke Please open a new pr.
